addons/account/static/src/components/duplicate_bill_warninng/duplicate_bill_warning.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { getCurrency } from "@web/core/currency"
+
+
+class DuplicateBillWarning extends Component {
+    static template = "account.DuplicateBillWarning";
+    static props = {};
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async discard() {
+        if (this.record?.isInEdition) {
+            await this.record.discard();
+        }
+    }
+
+    async open_bills_tree_and_discard() {
+        await this.discard();
+        const action = await this.orm.call("account.move", "open_duplicated_ref_bill_view", [this.props.record.resId], {});
+        this.action.doAction(action);
+    }
+
+
+    async open_bill_form_and_discard(id) {
+        await this.discard();
+        this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: "account.move",
+            res_id: id,
+            views: [[false, "form"]],
+            view_mode: "form",
+        });
+    }
+
+    get duplicate_bills_count() {
+        return this.props.record.data.duplicated_ref_ids.count;
+    }
+
+    get duplicate_bills_data() {
+        return this.props.record.data.duplicated_ref_ids.records.map((bill) => {
+            return {
+                name: bill.data.state == "draft" ? "Draft" : bill.data.name,
+                id: bill.resId,
+                partner_name: bill.data.partner_id[1],
+                total: `${bill.data.amount_total} ${getCurrency(bill.data.currency_id[0]).symbol}`
+            }
+        });
+    }
+}
+
+DuplicateBillWarning.template = "account.DuplicateBillWarning";
+registry.category("fields").add("duplicate_bill_warning", {
+    component: DuplicateBillWarning,
+    relatedFields: [
+        { name: "name", type: "char" },
+        { name: "currency_id", type: "many2one" },
+        { name: "partner_id", type: "many2one" },
+        { name: "amount_total", type: "monetary"},
+        { name: "state", type: "char"}
+    ],
+});
```

Comment:
Please configure your editor to always add a EOL at the end of the file (and fix all the files in the PR)

Reply:
aright, i tought it was set, bizzare